opencsp/app/sofast/lib/image_processing.py

```diff
@@ -462,8 +465,35 @@ def detect_blobs(image: np.ndarray, params: cv.SimpleBlobDetector_Params) -> Vxy
     return Vxy(np.array(pts).T)
 
 
+def detect_blobs_inverse(image: np.ndarray, params: cv.SimpleBlobDetector_Params) -> Vxy:
+    """Detect blobs in image. Blobs are defined as local light regions in
+    neighboring dark background.
+
+    NOTE: This definition of blobs is the inverse as in `image_processing.detect_blobs()`
+
+    Parameters
+    ----------
+    image : np.ndarray
+        Input image, uint8

```

Comment:
Presumably this should be a 2d image with a single color channel? AKA:

```2D input image, single color channel, NxM or NxMx1, uint8```

Reply:
Good idea! Changed!